src/form/Form.jsx

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types'
 import React from 'react'
 import ImmutablePropTypes from 'react-immutable-proptypes'
 
-import FormValueScope from './FormValueScope'
+export const FormScopeValueContext = React.createContext()
```

Comment:
I think it should be called `FormValueScopeContext` (as before).
Can you please rename this?

Reply:
done :)